pkgs/development/tools/build-managers/gradle/setup-hook.sh

```diff
@@ -54,7 +54,11 @@ gradleUpdateScript() {
     runHook preBuild
     runHook preGradleUpdate
 
-    gradle ${enableParallelUpdating:+--parallel} ${gradleUpdateTask:-nixDownloadDeps}
+    local verificationFlags=sha256
```

Comment:
there should probably be a small comment explaining that this is for fetching all dependencies and not for integrity checking (and is there any reason to use `pgp,sha256` here?)